### PR TITLE
ArmA3_Wasteland.Tavi v.15

### PR DESCRIPTION
**Pull request by Masiin**
TOPArma A3Wasteland series built for Arma 2 map Taviana
Initial Release - 13 April 2015

UPDATE v0.15 - 18 APRIL 2015
Fixed - Sector B guards spawning with obscene amount of rockets (Seabase)
Fixed - Vehicle stores fixed, vehicles should no longer blow up when bought (Masin)
Fixed - Sabina territory too small for location (Masin)
Fixed - ATM Locations and bearings to fit against buildings (Masin)
Fixed - Various Store Fixes and debugging (Masin)

Added - New territory capture points at Nuclear Power Plant (Masin)
Added - New Capture Radar Truck mission (Motavar)
Added - More mission spawn points (Masin)
Added - 3 New Side missions (Masin)

Changed - Mission loot brought down to balanced levels with Sector B (Seabase)
Changed - Bilgrad Gun Store location (Masin)
Changed - Gun store interaction distance increased to 4m from 3m (Motavar)

Removed - Kajmans and Blackfoots from Hostile Helicopter missions (Seabase)

### CODE REVIEW

File: client/functions/clientCompile.sqf [hunk 1]
```diff
@@ -77,8 +83,7 @@ fn_cameracheck = "addons\cameracheck\cameracheck.sqf" call mf_compile;
 timeSync = "client\functions\clientTimeSync.sqf" call mf_compile;
 
 // Update scripts
-updateTeamKiller = "client\functions\updateTeamKiller.sqf" call mf_compile;
```
Comment (l0dac) [inline client/functions/clientCompile.sqf:80]: Why is this removed?

File: server/functions/setupStoreNPC.sqf [hunk 2]
```diff
@@ -1,329 +1,389 @@
...
+					if ((getPosATL _sellBox) vectorDistance _boxPos > 0.1 ||
+					   {(vectorDir _sellBox) vectorDistance _boxVecDir > 0.1} ||
+					   {(vectorUp _sellBox) vectorDistance _boxVecUp > 0.1}) then
+					{
+						_sellBox setPosATL _boxPos;
+						_sellBox setVectorDirAndUp [_boxVecDir, _boxVecUp];
+					};
+				};
+			};
+*/
```
Comment (AgentRev) [inline server/functions/setupStoreNPC.sqf:384]: "lol lets coment everythin cuz i dun kno wat it does!!1!"
